src/inequality.jl

```diff
@@ -0,0 +1,129 @@
+"""
+$(TYPEDEF)
+
+An inequality relationship between two expressions.
+
+# Fields
+$(FIELDS)
+"""
+struct Inequality
+    """The expression on the left-hand side of the inequality."""
+    lhs
+    """The expression on the right-hand side of the inequality."""
+    rhs
+    """The relational operator of the inequality."""
+    relational_op
+    function Inequality(lhs, rhs, relational_op)
+        new(Symbolics.value(lhs), Symbolics.value(rhs), relational_op)
+    end
+end
+
+Base.:(==)(a::Inequality, b::Inequality) = all([isequal(a.lhs, b.lhs), isequal(a.rhs, b.rhs), isequal(a.relational_op, b.relational_op)])
+Base.hash(a::Inequality, salt::UInt) = hash(a.lhs, hash(a.rhs, hash(a.relational_op, salt)))
+
+@enum RelationalOperator leq geq # strict less than or strict greater than are not supported by any solver
+
+function scalarize(ineq::Inequality)
+    if ineq.relational_op == leq
+        scalarize(ineq.lhs) ≲ scalarize(ineq.rhs)
+    else
+        scalarize(ineq.lhs) ≳ scalarize(ineq.rhs)
+    end
+end
+
+function Base.show(io::IO, ineq::Inequality)
+    print(io, ineq.lhs, ineq.relational_op == leq ? " ≲ " : " ≳ ", ineq.rhs)
+end
+
+"""
+$(TYPEDSIGNATURES)
+
+Create an [`Inequality`](@ref) out of two [`Num`](@ref) instances, or an
+`Num` and a `Number`.
+
+# Examples
+
+```jldoctest
+julia> using Symbolics
+
+julia> @variables x y;
+
+julia> x ≲ y
+x ≲ y
+
+julia> x - y ≲ 0
+x - y ≲ 0
+```
+"""
+function ≲(lhs, rhs) 
```

Comment:
Could you add a one-line description for how to type `≲`? For example, [`Base.MathConstants.pi`](https://docs.julialang.org/en/v1.8/base/numbers/#Base.MathConstants.pi).

Symbolics.jl is not in SciML, but SciMLStyle doesn't allow Unicode to be used in public APIs.
See [SciMLStyle: General Naming Principles](https://github.com/SciML/SciMLStyle#general-naming-principles).
So it is probably better to change the function name.

Reply:
See  #787 

> So it is probably better to change the function name.

Yeah, but to what?